tool/grind.dart

```diff
@@ -16,15 +15,15 @@ import 'package:yaml/yaml.dart';
 
 /// The version of Dart Sass.
 final String _version =
-    loadYaml(new File('pubspec.yaml').readAsStringSync())['version'];
+    loadYaml(new File('pubspec.yaml').readAsStringSync())['version'] as String;
```

Comment:
It'll be quicker for me to review this PR if you keep the changes focused on `package:` URLs.